beta/nncf/tensorflow/api/compression.py

```diff
@@ -104,4 +97,11 @@ def build_controller(self, model: ModelType) -> TFCompressionAlgorithmController
             algorithm-specific compression during fine-tuning.
         :return: The instance of the `CompressionAlgorithmController`.
         """
-        return TFCompressionAlgorithmController(model)
+
+
+class TFZeroCompressionLoss(CompressionLoss):
```

Comment:
It looks like the wrong module for `TFZeroCompressionLoss`.

Reply:
Done.